api/routers/areas.py

```diff
@@ -74,6 +78,8 @@ async def create_area(new_area: AreaConfigDTO, reboot_processor: Optional[bool]
     """
     Adds a new area to the processor.
     """
+    # TODO: We have to autogenerate the ID.
+    # TODO: Test what happens when we send invalid occupancy rules, for example: [{}] (server explodes)
```

Comment:
Maybe this happens because the fields are Optional. I think none of them is optional though so maybe removing that is the answer

Reply:
It seems they are not optional so 🤷🏻‍♂️